benefits/static/css/styles.css

```diff
@@ -308,10 +308,11 @@ footer .footer-links li a:visited {
 
 /* Custom button: Sign Out */
 
-.signout-row {
+.signout-row,
+.previous-page-row {
```

Comment:
Oops 🙏 Just checked the sign out link for regressions, and I realized I caused a regression here. Instead of having `signout-row` and `previous-page-row` use the same code here, I'd just create a separate class declaration for `previous-page-row` so this PR isn't affecting signout-row at all. Sign out button can be refactored separately from this PR.